src/extension.ts

```diff
@@ -26,207 +31,225 @@ export class NewComment implements vscode.Comment {
     this.savedBody = this.body;
     this.label = " ";
   }
+  public static async showCommentThread(path: vscode.Uri | undefined = undefined) {
+    try {
+      const currentFilePath = path?.path || currentActiveFile?.path || vscode.window.activeTextEditor?.document.uri.path;
+      if (currentFilePath) {
+        const workspaceFolderName = vscode.workspace.workspaceFolders !== undefined ? vscode.workspace.workspaceFolders[0].name : "";
+        const pathSplit = currentFilePath.split(workspaceFolderName);
+        pathSplit[1] = "/.docs" + pathSplit[1] + ".json";
+        const jsonPath = pathSplit.join(workspaceFolderName);
+        const document = await vscode.workspace.openTextDocument(currentFilePath);
+        const comments = JSON.parse((await fs.promises.readFile(jsonPath)).toString()) || {};
+        for (const key in comments) {
+          const value = comments[key];
+          const lineNumber = +key.split('-')[0];
+          const lineTextInDoc = document.lineAt(lineNumber-1).text;
+          const lineTextInJson = key.split('-')[1];
+          if (lineTextInJson === '' && lineTextInDoc !== lineTextInJson) {
+          continue;
+          }
+          const uri = <vscode.Uri>path || currentActiveFile || vscode.window.activeTextEditor?.document.uri;
+          const commentThread = commentController.createCommentThread(uri, new vscode.Range(lineNumber - 1, 0, lineNumber - 1, 0), []);
+          commentThread.canReply = false;
+          commentThread.collapsibleState = vscode.CommentThreadCollapsibleState.Collapsed;
+          const myNewComment = new NewComment(value, lineNumber, vscode.CommentMode.Editing,
+            { name: "" },
+            commentThread,
+            "what's a context value!"

```

Comment:
If this is not required, could we replace it with an empty string?

Reply:
done